data/Makefile.am

```diff
@@ -33,8 +36,16 @@ desktop_DATA =						\
 	setup-shell.desktop				\
 	$(NULL)
 
+demovideodir = $(datadir)/eos-demo-mode
+demovideo_DATA = \
+	eos-demo-video.ogv \
+	eos-demo-video.desktop \
```

Comment:
We could do that, though in my view installing it into the demo session's home directory seems like a cleaner approach that achieves the same result. What was the reason for doing it this way?